.github/Issue_Template/bug.yml

```diff
@@ -0,0 +1,53 @@
+name: ​🐞 Bug
+description: Report an issue to help us improve the project.
+title: "[BUG] <write a small description here>"
+labels: ["bug", "goal: fix"]
+body:
+  - type: textarea
+    attributes:  
+      label: Description   
+      id: description  
+      description: A brief description of the issue or bug you are facing, also include what you tried and what didn't work.
+    validations:
+      required: false
+  - type: textarea
+    attributes: 
+      label: Screenshots
+      id: screenshots
+      description: Please add screenshots if applicable
+    validations:
+      required: false
+  - type: textarea
+    attributes: 
+      label: Any additional information? 
+      id: extrainfo
+      description: Any additional information or Is there anything we should know about this bug? 
+    validations: 
+      required: false
+  - type: dropdown
+    id: browsers
+    attributes: 
+      label: What browser are you seeing the problem on? 
```

Comment:
There's an issue with this. Refer to my change requests in your earlier PR for the bug template.

Reply:
I have made the changes in other pr